client/public/src/components/details.jsx

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
```

Comment:
you probably don't need Component in this or any of the functional components

Reply:
good catch!